src/test/consensus/DistributedValidatorsSim_test.cpp

```diff
@@ -87,15 +90,17 @@ class DistributedValidators_test : public beast::unit_test::suite
 
         // Initialize persistent collector logs specific to this method
         std::string const prefix =
-                "DistributedValidators_"
-                "completeTrustCompleteConnectFixedDelay";
+                "DistributedValidatorsSim_"
+                "completeTrustCompleteConnectNormalDelay";
         std::fstream
                 txLog(prefix + "_tx.csv", std::ofstream::app),
                 ledgerLog(prefix + "_ledger.csv", std::ofstream::app);
 
         // title
+        mutex.lock();
```

Comment:
Consider using `lock_guard` instead of explicit lock and unlock throughout this simulation.

Reply:
I looked into `lock_guard`s before. I removed them because I had them in their own scope and so everything was indented, and I didn't like that for purely ascetic reasons. But that's not really a justification I suppose. I'll add the `lock_guard`s.